installers/metrics/ansible/roles/pgo-metrics/tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Set Metrics Directory Fact
   set_fact:
-    metrics_dir: "{{ ansible_env.HOME }}/.pgo/metrics/{{ metrics_namespace }}"
```

Comment:
Why are we getting rid of the `metrics_namespace` section of the path?

Reply:
I did this to simplify the location of where we are looking for the files within the `pgo-deployer`, as we don't necessarily know the namespace at that moment. However, I understand we need to balance that with the native Ansible installer. Perhaps that is overly aggressive?

Reply:
Ah i see. We won't know what the namespace would be to set in the dockerfile. Perhaps we could add an env var with the path that points to the pgmonitor files?  Then check that env var and if unset try to download using the url.